packages/loader/container-loader/src/serializedStateManager.ts

```diff
@@ -432,9 +428,9 @@ export class SerializedStateManager {
 			{
 				eventName: "getPendingLocalState",
 				details: {
-					notifyImminentClosure: props.notifyImminentClosure,
-					sessionExpiryTimerStarted: props.sessionExpiryTimerStarted,
-					snapshotSequenceNumber: props.snapshotSequenceNumber,
+					notifyImminentClosure: false,
+					sessionExpiryTimerStarted: undefined,
+					snapshotSequenceNumber: undefined,
```

Comment:
i think these should stay

Reply:
sessionExpiryTimerStarted and snapshotSequenceNumber have never been passed here, even before this change.  I was actually thinking about removing them outright rather than leaving them here and undefined, but wasn't sure what impact it'd have on telemetry.

Reply:
makes sense since both fields are encapsulated in the pending runtime state